netadapter/server/grpcserver/grpcserver.go

```diff
@@ -48,7 +48,9 @@ func (s *gRPCServer) Connections() []server.Connection {
 	panic("unimplemented")
 }
 
-type gRPCConnection struct{}
+type gRPCConnection struct {
+	onDisconnectedHandler server.OnDisconnectedHandler
```

Comment:
Just to make sure: I'm expected to call onDisconnectedHandler when the other side disconnects - right?
Should I also call it when .Disconnect is called()?

Reply:
Yep!